Common/include/toolboxes/MMS/CIncTGVSolution.hpp

```diff
@@ -92,5 +92,5 @@ class CIncTGVSolution: public CVerificationSolution {
    */
   void GetBCState(const su2double *val_coords,
                   const su2double val_t,
-                  su2double       *val_solution);
+                  su2double       *val_solution) const override;
```

Comment:
As mentioned in #824 I added the `const` and `override/final` specifiers to the MMS classes, `const` methods cannot alter the state of the class and so the "Get's" should be ok to call in parallel.